Godeps

```diff
@@ -46,6 +46,7 @@ github.com/wvanbergen/kafka 1a8639a45164fcc245d5c7b4bd3ccfbd1a0ffbf3
 github.com/wvanbergen/kazoo-go 0f768712ae6f76454f987c3356177e138df258f8
 golang.org/x/crypto 3760e016850398b85094c4c99e955b8c3dea5711
 golang.org/x/net 99ca920b6037ef77af8a11297150f7f0d8f4ef80
+golang.org/x/text cf4986612c83df6c55578ba198316d1684a9a287
```

Comment:
This package is need to parse the XML with a a charset reader. The XML output has an attribute `encoding=ISO88591` and I cannot simply parse it with `xml.Unmarshal` because go lag complains about it like this:

```
xml: encoding "iso8859-1" declared but Decoder.CharsetReader is nil
```

However, If I simply remove that attribute from XML then it works just fine. I'm not sure if we should just stripped out that attribute to remove this dependency